Lib/test/pickletester.py

```diff
@@ -2152,7 +2158,7 @@ def remove_frames(pickled, keep_frame=None):
 
         frame_size = self.FRAME_SIZE_TARGET
         num_frames = 20
-        obj = [bytes([i]) * frame_size for i in range(num_frames)]
+        obj = {i: bytes([i]) * frame_size for i in range(num_frames)}
```

Comment:
IIUC, small integers don't produce a frame? Can you add a comment?

Reply:
I may misunderstand, actually...

Reply:
I have added small objects (dict keys) between large objects.